spec/factories/barcode_items.rb

```diff
@@ -14,9 +14,13 @@
 FactoryGirl.define do
```

Comment:
If you merge in master this will probably update to FactoryBot. Other than this, everything looks good!

Reply:
Just so I'm clear -- are you saying I should merge in master *first* or is this just an advisory that it will change when it's merged?

Reply:
Just an advisory :)